src/components/ArtObjectPageComponents/PanelDetails/index.jsx

```diff
@@ -182,49 +191,70 @@ class Image extends Component {
       isLoaded,
       object,
       activeImageIndex,
-      setActiveImageIndex
+      setActiveImageIndex,
+      renditions
     } = this.props;
+
+    // This indicates that there was an error with rendering the Zoom component
     const { didCatchFailure } = this.state;
-    
+    const showZoomImageView = Boolean(!didCatchFailure && object.id && activeImageIndex === 0);
+    const renditionsExist = renditions?.length > 0;
+
     let additionalStyle = {};
-    if (!didCatchFailure) {
+    let imageUrlToRender = '';
+
+    // If we encountered failure during rendering of the Zoom component, we'll hide it
+    // Additionally, if the user clicked on an image from the rendition thumbnails, we'll render it instead
+    if (!didCatchFailure && activeImageIndex === 0) {
       additionalStyle = { ...additionalStyle, display: 'none' };
     };
 
+    // We'll render the image from the object itself 
+    if (activeImageIndex === 0) {
+      imageUrlToRender = object.imageUrlLarge
+    }  // Otherwise, one of the renditions was clicked. So we need to render that image
+    else {
+      const imageThumbnailPreview = renditions[activeImageIndex].proxies.find((proxy) => {
+        return proxy.name === 'Preview'
+      });
+      imageUrlToRender = `https://barnesfoundation.netx.net${imageThumbnailPreview.file.url}/`;
+    }
```

Comment:
If we're on the first image index, it means we should render the URL from the object itself. If the image index is greater than 0, it means user clicked on a thumbnail image so we should render that preview URL instead.